src/plugins/func/funcUtils.ts

```diff
@@ -52,7 +52,7 @@ module.exports.handler = async function (context, req) {
 
   private static defaultFunctionSlsObject(name: string) {
     return {
-      handler: "index.handler",
```

Comment:
no, it is relative to the project root.  So it would be like:
```
functionApp
--hello
----function.json
--src
----handlers
------{name}.js (with handler function inside)
```